src/projects/automation/datastores/tables/table_relationships.md

```diff
@@ -45,3 +46,5 @@ If you had a table of movies, a table of categories and a table of actors you mi
 - Movies 1:Many Actors
```

Comment:
I know this is not a change in this PR, but this isn't going to be an easy fix, so probably better avoid using `:Something`